shared/chat/conversation/messages/text.desktop.js

```diff
@@ -111,12 +112,22 @@ const _avatarStyle = {
   marginRight: globalMargins.tiny,
 }
 
-export default withHandlers({
-  onIconClick: (props: Props) => event => {
-    props.onAction(props.message, event)
-  },
-})(_MessageTextComponent)
-
 const stylesFirstNewMessage = {
   borderTop: `solid 1px ${globalColors.orange}`,
 }
+
+export default compose(
+  shouldUpdate((props: Props, nextProps: Props) => {
+    return !shallowEqual(props, nextProps, (obj, oth, key) => {
+      if (key === 'style') {
+        return shallowEqual(obj, oth)
+      }
+      return undefined
+    })
+  }),
+  withHandlers({
```

Comment:
I moved this around in https://github.com/keybase/client/pull/5183. There's a shared message component for common styling between attachments and text